gerritbot/examples/gerritbot-console.rs

```diff
@@ -95,8 +95,16 @@ fn main() {
         }
     };
     let email = args.email.clone();
-    let stdin_lines = tokio::io::lines(BufReader::new(tokio::io::stdin()))
-        .map_err(|e| error!("failed to read line from stdin: {}", e));
+    let (stdin_lines_sender, stdin_lines) = futures::sync::mpsc::channel::<String>(1);
```

Comment:
Don't you want to import this long name?

Reply:
Fixed in 9bd7e9b064bba42f1e98c737a934996f96d74d49.